package.json

```diff
@@ -9,13 +9,13 @@
     "build": "yarn run build-js && yarn run build-css",
     "build-css": "node-sass --include-path node_modules static/sass --output static/css && postcss --use autoprefixer --replace 'static/css/**/*.css' && postcss --use cssnano --dir static/minified 'static/css/**/*.css'",
     "build-js": "yarn run copy-3rd-party-js && yarn run build-js-transpile",
-    "copy-3rd-party-js": "cp node_modules/d3/build/d3.min.js static/js/modules && cp node_modules/d3-geo/build/d3-geo.min.js static/js/modules && cp node_modules/topojson-client/dist/topojson-client.min.js static/js/modules && cp node_modules/billboard.js/dist/billboard.min.js static/js/modules && cp node_modules/moment/min/moment.min.js static/js/modules && cp node_modules/clipboard/dist/clipboard.min.js static/js/modules && cp node_modules/raven-js/dist/raven.min.js static/js/modules",
+    "copy-3rd-party-js": "cp node_modules/clipboard/dist/clipboard.min.js static/js/modules/ && cp node_modules/raven-js/dist/raven.min.js static/js/modules/",
     "build-js-transpile": "rollup -c",
     "serve": "./entrypoint.sh 0.0.0.0:${PORT}",
     "watch": "watch -p 'static/sass/**/*.scss' -p 'static/js/**/*.js' -c 'yarn run build'",
     "watch-scss": "watch -p 'static/sass/**/*.scss' -c 'yarn run build-css'",
     "watch-js": "watch -p 'static/js/**/*.js' -c 'yarn run build-js'",
-    "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite _site/ build/ .jekyll-metadata .bundle"
+    "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite _site/ build/ .jekyll-metadata .bundle static/js/modules/* static/js/dist/*"
```

Comment:
I'm not sure about adding `static/js/modules` to clean, especially now when we removed it.

Reply:
Also, while we are getting rid of `modules` probably it should be removed from .gitignore as well

Reply:
There's still a couple of modules we need so maybe we don't need to clean it out on second thoughts, wdyt?

Reply:
Ah, true, didn't notice that we still keep some around. I'm fine with cleaning, so unnecessary files are removed, but I guess this script will remove the README.md that is in that folder now (and that's what happened in the PR I guess - that's why I thought it's not needed anymore).

Reply:
Anyway, keeping readme in this folder is probably unnecessary anyway, we could just put the whole folder into gitignore, so nothing gets added there.